### PR TITLE
Switch to containers for the majority of testing

### PR DESCRIPTION
**Pull request by loganharbour**

### CODE REVIEW

File: testroot [hunk 1]
```diff
@@ -0,0 +1,3 @@
+allow_warnings = false
```
Comment (GiudGiud) [inline testroot:1]: what testing will use this?
the allow_warnings = false is more restrictive than all the apps are

Comment (loganharbour) [reply testroot:1]: The opt testing. We can remove it, I don't mind

**Review by GiudGiud — APPROVED** [review]
looks great thanks for fixing the syntax

File: tests/simple_diffusion/simple_diffusion/gold/simple_diffusion_out.e
Comment (GiudGiud) [inline tests/simple_diffusion/simple_diffusion/gold/simple_diffusion_out.e:1]: Do you need this?

It does not really fit with the vtb vibe. If you need this let s hide it in apps/ci or something

Comment (loganharbour) [reply tests/simple_diffusion/simple_diffusion/gold/simple_diffusion_out.e:1]: This is just for testing on this PR

**Comment by loganharbour** [comment]
@GiudGiud this is good to go

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**